### ISSUE TITLE
(error) FAILED to execute command: "php"

### ISSUE THREAD

**Issue by khalilsarabey** [VemtoOrg/vemto2-issues#185]
**Error Message:**
(error) FAILED to execute command: "php" -d display_errors=on "/Applications/Vemto.app/Contents/Resources/static/VMTTL1"
    |> 
Warning: Undefined array key "order" in phar:///Applications/Vemto.app/Contents/Resources/static/VMTTL1/classes/TableRepository.php on line 59

Fatal error: Cannot use Illuminate\Http\Request as Request because the name is already in use in /Users/khalilsarabey/Sites/VirtuosiTravelSystem/app/Http/Controllers/Api/RequestController.php on line 6

    |> PHP Warning:  Undefined array key "order" in phar:///Applications/Vemto.app/Contents/Resources/static/VMTTL1/classes/TableRepository.php on line 59
PHP Fatal error:  Cannot use Illuminate\Http\Request as Request because the name is already in use in /Users/khalilsarabey/Sites/VirtuosiTravelSystem/app/Http/Controllers/Api/RequestController.php on line 6

**Stack:**
```
Error: Command failed: "php" -d display_errors=on "/Applications/Vemto.app/Contents/Resources/static/VMTTL1"
PHP Warning:  Undefined array key "order" in phar:///Applications/Vemto.app/Contents/Resources/static/VMTTL1/classes/TableRepository.php on line 59
PHP Fatal error:  Cannot use Illuminate\Http\Request as Request because the name is already in use in /Users/khalilsarabey/Sites/VirtuosiTravelSystem/app/Http/Controllers/Api/RequestController.php on line 6

    at ChildProcess.exithandler (node:child_process:407:12)
    at ChildProcess.emit (node:events:527:28)
    at maybeClose (node:internal/child_process:1092:16)
    at Socket.<anonymous> (node:internal/child_process:451:11)
    at Socket.emit (node:events:527:28)
    at Pipe.<anonymous> (node:net:709:12)
```

**How to reproduce:**
1. 
2. 
3. 

**Expected behavior:**

**Actual behavior:**

**Additional context:**

---

Vemto 2.1.0
darwin 23.5.0
Locale: en-GB
App version: 2.1.0

**Comment by TiagoSilvaPereira** [comment]
@khalilsarabey please try changing to the Database Schema reader mode, This bug happens on Migration Schema reader (I'll fix this soon)

**Comment by TiagoSilvaPereira** [comment]
Fixed on 2.1.4
